invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/AdminForm.js

```diff
@@ -81,12 +108,24 @@ export class AdminForm extends Component {
     this.setState({ error: undefined });
   };
 
+  transformInitialValues = (formData) => {
+    const { resourceSchema } = this.props;
+    return mapValues(formData, (value, key) => {
+      const fieldSchema = resourceSchema[key];
+      if (fieldSchema?.metadata?.type === "json" && typeof value === "object") {
+        return JSON.stringify(value);
```

Comment:
here you return a string representation of the json object and that is passed in formik's initialValues? How do you consume the stringified json object?

Reply:
in the TextAreaField, it shows the stringified json object in the text box, otherwise you get the object back and it just displays the type `[Object object]`